common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -257,4 +257,31 @@ public Properties cryptoConf() {
     return CryptoUtils.toCryptoConf("spark.network.crypto.config.", conf.getAll());
   }
 
+  /**
+   * When memory usage of Netty is above this water mark, it's regarded as memory shortage.
```

Comment:
This is inconsistent with the executor memory configuration. For executor memory, we have `spark.executor.memory` for heap size, and `spark.memory.offHeap.size` for off-heap size, and these 2 together is the total memory consumption for each executor process.

Now we have a single config for shuffle service total memory consumption, this seems better, shall we fix the executor memory config?